src/insert/comments-expanded-advert.ts

```diff
@@ -67,6 +106,30 @@ const createResizeObserver = (rightColumnNode: HTMLElement) => {
 	resizeObserver.observe(rightColumnNode);
 };
 
+const removeMobileCommentsExpandedAds = (): Promise<void> => {
+	const currentBreakpoint = getBreakpoint(getViewport().width);
+	if (currentBreakpoint === 'mobile') {
```

Comment:
Could we include tablet as well? We don't show a right column until the desktop breakpoint. Happy for this to be a follow-up ticket.

Reply:
The ticket just specified mobile for this work which is why we haven't implemented tablet. I think it makes more sense as a follow-up ticket